src/pages/export-page/ExportPage.tsx

```diff
@@ -28,12 +30,14 @@ export const ExportPage = () => {
 
   const timetableEventsDownloadLink = useMemo(() => {
     const timetableCSV = resolveTimetableEventsCSV(
-      timetableEvents,
+      statusChangeEvent
+        ? [...timetableEvents, statusChangeEvent as DevelopmentPlanTimetable]
```

Comment:
I don't really like this cast, it's basically asserting that an option field is present event though the type allows it to be undefined. I think it would be better if we could find a way to make sure we know we have the event when we get here.

Reply:
I changed it to check that the event's developmentPlanEvent is not undefined. Looks like the compiler is still complaining so I left the casting in

Reply:
Yeah, I was hoping we might be able to make the field not optional on the type but I think would need a bit of a redesign and I don't think it would be worth it at this point